scripts/zones/Windurst_Waters_[S]/npcs/Door_Acolyte_hostel_up.lua

```diff
@@ -0,0 +1,36 @@
+-----------------------------------
+-- Area: Windurst Waters (S)
+--  NPC: Door Acolyte hostel
+-- Type: Quest NPC
+-- !pos  146.619,-8.525,242.874 94
+-----------------------------------
+local ID = require("scripts/zones/Windurst_Waters_[S]/IDs")
+require("scripts/globals/keyitems")
+require("scripts/globals/missions")
+require("scripts/globals/settings")
+require("scripts/globals/quests")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player,npc)
+    local sayItWithAHandbag = player:getQuestStatus(CRYSTAL_WAR, dsp.quest.id.crystalWar.SAY_IT_WITH_A_HANDBAG)
```

Comment:
Same here

Reply:
My guess here is because the tabled name is long, so the assignment is to make the line for the if check shorter.

Reply:
@ibm2431   that is correct, but I can remove it. I'll go through the script and make changes to the guild locals too :)